Mlem/Views/Tabs/Feeds/Feed Types/AggregateFeedView.swift

```diff
@@ -76,7 +77,7 @@ struct AggregateFeedView: View {
                     switch selectedFeed {
                     case .all, .local, .subscribed:
                         await markReadBatcher.flush()
-                        await postTracker?.changeFeedType(to: .aggregateFeed(apiSource, type: selectedFeed.toApiListingType))
+                        await postTracker?.changeFeedType(to: .aggregateFeed(apiSource, type: selectedFeed.toAPIListingType))
```

Comment:
I think we need to standardise our naming conventions in cases like this - we use different styles in different places, particularly with `URL`. Is there official guidance anywhere for Swift specifically? I found this SA thread, which has some opinions on language-agnostic camelCase acronyms https://stackoverflow.com/questions/15526107/acronyms-in-camelcase

Reply:
I agree with lowercase acronyms, both for personal preference and because they align with the code generator. Unless I add *yet more* exception cases to the parsing, fields named things like `delete_url` are going to be camelized to `deleteUrl`. The all-caps `API` prefix is a holdover from the v1 codebase conventions, but it's a one-line change to use `Api` instead.